helm/superset/templates/_helpers.tpl

```diff
@@ -96,7 +96,7 @@ CACHE_CONFIG = {
 }
 DATA_CACHE_CONFIG = CACHE_CONFIG
 
-SQLALCHEMY_DATABASE_URI = f"postgresql+psycopg2://{env('DB_USER')}:{env('DB_PASS')}@{env('DB_HOST')}:{env('DB_PORT')}/{env('DB_NAME')}"
+SQLALCHEMY_DATABASE_URI = env('SQLALCHEMY_DATABASE_URI', f"postgresql+psycopg2://{env('DB_USER')}:{env('DB_PASS')}@{env('DB_HOST')}:{env('DB_PORT')}/{env('DB_NAME')}")
```

Comment:
Why are we not just overriding this in `superset_config.py`?

Reply:
If `superset_config.py` is loaded [here](https://github.com/apache/superset/blob/95325c4673b3bad887521530710783e2553f2ac9/helm/superset/templates/secret-superset-config.yaml#L32-L33)

```yaml
apiVersion: v1
kind: Secret
...
stringData:
  superset_config.py: |
    {{- include "superset-config" . | nindent 4 }}
```

and `superset-config` is defined [here](https://github.com/apache/superset/blob/95325c4673b3bad887521530710783e2553f2ac9/helm/superset/templates/_helpers.tpl#L65-L138)

```
{{- define "superset-config" }}
...
SQLALCHEMY_DATABASE_URI = f"postgresql+psycopg2://{env('DB_USER')}:{env('DB_PASS')}@{env('DB_HOST')}:{env('DB_PORT')}/{env('DB_NAME')}"
...
{{- end }}
```

Isn't that what we're essentially doing? That said, I hadn't noticed the existence of `.configOverrides` at the time, since Helm templates are kind of a disaster to comprehend.